Docs edits (#8076)
PrefectHQ/prefect: docs/faq.md

Prefect 2 is licensed under the [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0), an [OSI approved](https://opensource.org/licenses/Apache-2.0) open-source license. If you have any questions about licensing, please [contact us](mailto:hello@prefect.io).

> ### Is the Prefect v2 Cloud URL different than the Prefect v1 Cloud URL?  

Reviewer: ```suggestion
### Is the Prefect 2 Cloud URL different than the Prefect 1 Cloud URL?  
```

> Yes. Prefect Cloud for v2 is at [app.prefect.cloud/](https://app.prefect.cloud) while Prefect Cloud for v1 is at [cloud.prefect.io](https://cloud.prefect.io/).

Reviewer: ```suggestion
Yes. 

Prefect Cloud 2 is at [https://app.prefect.cloud/](https://app.prefect.cloud).

Prefect Cloud 1 is at [https://cloud.prefect.io](https://cloud.prefect.io/).
```

### Can I enforce ordering between tasks that don't share data?

> Yes! For more information, see the [`Tasks` section](/concepts/tasks/#wait-for).

Reviewer: ```suggestion
Yes! For more information, see the [Tasks section](/concepts/tasks/#wait-for).
```

Reviewer: Not sure why that was styled as code.

Prefect does not have any additional requirements besides those installed by `pip install --pre prefect`. The entire system, including the UI and services, can be run in a single process via `prefect orion start` and does not require Docker.

Prefect Cloud users do not need to worry about the Prefect database. Prefect Cloud uses PostgreSQL on GCP behind the scenes. To use PostgreSQL with a self-hosted Orion server, users must provide the [connection string][prefect.settings.PREFECT_ORION_DATABASE_CONNECTION_URL] for a running database via the `PREFECT_ORION_DATABASE_CONNECTION_URL` environment variable.

### What databases does Prefect support?

> A self-hosted Prefect Orion server can work with SQLite and PostgreSQL. New Prefect installs default to a SQLite database hosted at `~/.prefect/orion.db` on Mac or Linux machines. SQLite and PostgreSQL are not installed by Prefect.

Reviewer: ```suggestion
A self-hosted Prefect Orion server can work with SQLite or PostgreSQL. New Prefect installs default to a SQLite database hosted at `~/.prefect/orion.db` on Mac or Linux machines. SQLite and PostgreSQL are not installed by Prefect.
```
